SeaPublicWebsite/Program.cs

```diff
@@ -21,6 +22,8 @@ public static void Main(string[] args)
             
             var builder = WebApplication.CreateBuilder(args);
             
+            builder.WebHost.ConfigureKestrel(serverOptions => serverOptions.AddServerHeader = false);

```

Comment:
LGTM, worth sticking in a citation for why this is being removed? or just a comment "Hide what server we are using for security reasons"